src/ResourceManager/Sql/Commands.Sql/VirtualNetworkRule/Cmdlet/RemoveAzureSqlServerVirtualNetworkRule.cs

```diff
@@ -0,0 +1,89 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using System.Collections.Generic;
+using System.Globalization;
+using System.Management.Automation;
+
+namespace Microsoft.Azure.Commands.Sql.VirtualNetworkRule.Cmdlet
+{
+    /// <summary>
+    /// Defines the Remove-AzureRmSqlServerVirtualNetworkRule cmdlet
+    /// </summary>
+    [Cmdlet(VerbsCommon.Remove, "AzureRmSqlServerVirtualNetworkRule", SupportsShouldProcess = true, ConfirmImpact = ConfirmImpact.Medium)]
```

Comment:
@kisantia we should add a separate parameter set to allow users to pass the `VirtualNetworkRule` object that they want to remove to this cmdlet. This will enable the piping scenario `Get-* | Remove-*`. Please see our guide on piping for more information: https://github.com/Azure/azure-powershell/wiki/Piping-in-PowerShell

Reply:
Why do we need this? The same parameters need to be given for both the Get and Remove cmdlets.